packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -26,17 +26,24 @@ import ContextMenu from './context-menu';
 import StylesPreview from './preview';
 
 function ScreenRoot() {
-	const { variations } = useSelect( ( select ) => {
+	const { variations, canEditCSS } = useSelect( ( select ) => {
+		const {
+			getEntityRecord,
+			__experimentalGetCurrentGlobalStylesId,
+			__experimentalGetCurrentThemeGlobalStylesVariations,
+		} = select( coreStore );
+
+		const globalStylesId = __experimentalGetCurrentGlobalStylesId();
+		const globalStyles = globalStylesId
+			? getEntityRecord( 'root', 'globalStyles', globalStylesId )
+			: undefined;
+
 		return {
-			variations:
-				select(
-					coreStore
-				).__experimentalGetCurrentThemeGlobalStylesVariations(),
+			variations: __experimentalGetCurrentThemeGlobalStylesVariations(),
+			canEditCSS: globalStyles?._links?.[ 'wp:action-edit-css' ] ?? false,
```

Comment:
I find this check a bit confusing since it's not immediately obvious that the presence of a link should determinate the ability to customize CSS.

I think the approach taken in https://github.com/WordPress/gutenberg/pull/46666 where there is a new explicit setting is more clear.

Reply:
This is a pretty standard way to check the capabilities of a user - https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%27wp%3A+path%3A%2F%5Epackages%5C%2F%2F&type=code&l=packages%2F&p=1.

The `canUserUseUnfilteredHTML` selector uses a similar method.

Reply:
I agree with @Mamaduka that it'd be good to follow the approach used elsewhere.

It'd be good to coerce this value into a boolean though, at the moment `canEditCSS` ends up as an array instead of `true`.

Reply:
Thanks for catching this, Dan!

Resolved in [760cb95](https://github.com/WordPress/gutenberg/pull/46815/commits/760cb953f1e8583a4c34ff7c61f0a20ba6ef650c).

Reply:
> This is a pretty standard way to check the capabilities of a user

TIL! Thanks for clarifying.